klippy/extras/output_pin.py

```diff
@@ -22,30 +23,41 @@ def __init__(self, config):
             self.mcu_pin = ppins.setup_pin('digital_out', config.get('pin'))
             self.scale = 1.
             self.last_cycle_time = self.default_cycle_time = 0.
-        self.mcu_pin.setup_max_duration(0.)
         self.last_print_time = 0.
         static_value = config.getfloat('static_value', None,
                                        minval=0., maxval=self.scale)
         if static_value is not None:
+            self.mcu_pin.setup_max_duration(0.)
             self.last_value = static_value / self.scale
             self.mcu_pin.setup_start_value(
                 self.last_value, self.last_value, True)
         else:
+            self.reactor = self.printer.get_reactor()
+            self.host_ack_timeout = config.getfloat('host_acknowledge_timeout',
+                                                    0, minval=0.)
+            #ensure that safety timeout is big enough for comm. latency
+            if self.host_ack_timeout > 0:
+                self.host_ack_timeout = max(self.host_ack_timeout, 0.500)
```

Comment:
> `minval=max(self.host_ack_timeout, 0.500)`
Wouldn't this disallow zero? Currently, zero indicates "no timeout".

> it would give the host 100ms to respond to the event 
For this, the `relax_time` was supposed. But no problem there.

Reply:
The minmax checks do not apply to the default.  So, it wont stop the default of zero - though it would prevent a zero in the config - but that seems okay to me.

-Kevin